salt/modules/win_dns_client.py

```diff
@@ -22,15 +24,25 @@ def __virtual__():
     '''
     Only works on Windows systems
     '''
-    if salt.utils.platform.is_windows():
-        return 'win_dns_client'
-    return (False, "Module win_dns_client: module only works on Windows systems")
+    if not salt.utils.platform.is_windows():
+        return False, 'Module win_dns_client: module only works on Windows ' \
+                      'systems'
+    if not HAS_LIBS:
+        return False, 'Module win_dns_client: missing required libraries'
+    return 'win_dns_client'
```

Comment:
You could at least test that the `__virtual__` function returns the correct things when it's windows (or not) and has libs (or not)

Reply:
Added some tests... hope you like them